internal/web/jobstorage/job_storage.go

```diff
@@ -68,6 +68,7 @@ type Artifacts struct {
 type JobData struct {
 	Variant                string    `json:"variant"`
 	Model                  string    `json:"model"`
+	Architecture           string    `json:"architecture"`
```

Comment:
important addition hehe 😅

Reply:
yes :D . It's eventually used as a `--platform` argument on the docker command. I'm not 100% sure this is the was we use it in our pipelines. Is this how auroraboot infers the architecture or is there an explicit argument I should pass? I need to check.